docs/en/build-store/refresh-control.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Validations and refresh control
+table_of_contents: true
+---
+
+# Validations and refresh control
+
+## What is refresh control?
+
+_“🎵 baby don't update me, don't update me no more 🎵”_
+
+
+Refresh control is a mechanism for controlling the revisions of a snap that will be offered as updates  to a device. A *gating snap* can control the revisions of a *gated snap* so that, if the *gating snap* is installed on a device, only revisions of the *gated snap* which have been validated by the publisher of the *gating snap* can be installed or refreshed to.
+
+A gating snap can control revisions of one or more gated snaps. Similarly, a gated snap can require verification from more than one gating snap, and a particular revision of the gated snap will only be installable or refreshable to, if it satisfies the constraints of all its gating snaps. However, because it can be confusing to device operators, use of multiple gating snaps to control one specific gated snap is not recommended. The best practice is to use a single gating snap for each gated snap.
+
+This is especially useful for device manufacturers to mediate releases of the *core* snap, allowing time for testing and validation of new revisions on their device models and compatibility with their software.  Note this is separate from the ability for device managers to control the schedule of refreshes and revision control via a snap proxy.
+
+
+## Preliminary setup
+
+Establishing the relationship between a gating and gated snap is required only once and can only be performed by a user with reviewer permission for the gating snap's store.
+
+Ask a store administrator/reviewer to edit the package declaration for the gating snap on the [snap's page][1]. The administrator should:
+
+* Click on "review capabilities"
+* Under "refresh control" put a json list with the snap IDs of the snaps you want gated: \["xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx", ...\]
+* The list can, of course, contain more than one snap ID, in which case this gating snap will be able to control updates for all the gated snaps.
+
+## Publisher setup
+
+Since gating involves pushing [`validation` assertions][2] which are signed by the publisher indicating their authorization to restrict updates of snaps, some GPG keys must be generated and published to the store. If a store key already exists, it can be used. If not, one must be generated and registered.
+
+1. `snapcraft create-key` to create a key. (by default it's named "default" but a key name can be given to create a custom one so e.g. each snap is signed with its own key). Do not forget the passphrase used to secure the key or misplace the key files.
```

Comment:
I am pretty sure there are already docs that show how to generate and register keys. Consider replacing this with a url to the definitive docs since this same content should almost certainly not be duplicated everywhere it is needed since that becomes a maintenance burden and confuses readers.

Reply:
I prefer keeping this as a self-contained resource that doesn't unnecessarily branch the reader out (Wikipedia effect). I agree to linking to an existing resource for reference but think we should keep this mini-tutorial to present a complete walkthrough. As usual, feel free to change it later.